pages/about.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Document</title>
+<link href="../styles/style.css" rel="stylesheet" />
 </head>
+
 <body>
-  
+    <div class="container">
+        <div class="header">
+        
+        </div>
+        <div class="sidebar">
+            <img class="img-side" src="../images/side.jpeg" alt="my image">
+        </div>
+        <div class="content">
+            <p>
+                <br>
+                <br>
+<h1> A little about me:</h1>
+<h3>My name is Intesar Siraj and I am on the road to become a software developer.<br>
```

Comment:
The tags that start with `h` are heading tags. They should only be used for headings, not for full paragraphs of text. This is where `p` should be used instead.